types/jquery/index.d.ts

```diff
@@ -146,12 +146,22 @@ interface JQueryStatic {
      * Return a collection of matched elements either found in the DOM based on passed argument(s) or created
      * by passing an HTML string.
      *
-     * @param element_elementArray A DOM element to wrap in a jQuery object.
-     *                             An array containing a set of DOM elements to wrap in a jQuery object.
+     * @param element A DOM element to wrap in a jQuery object.
      * @see \`{@link https://api.jquery.com/jQuery/ }\`
      * @since 1.0
      */
-    <T extends Element>(element_elementArray: T | ArrayLike<T>): JQuery<T>;
+    // tslint:disable-next-line:unified-signatures
```

Comment:
Could you add a comment above the `tslint:disable`s indicating that this is a workaround for a TypeScript 2.4 bug?